docs/modules/deployment/pages/time-series-storage/timeseries/time-series-storage.adoc

```diff
@@ -65,9 +68,11 @@ opennms-newts
 [source, console]
 sudo systemctl restart opennms
 
-== Write to Time Series Integration Plugin in parallel
+[[ga-dual-write-integration]]
+=== Write to Time Series Integration Plugin in parallel
 
-You can enable writing to any time series integration plugin in parallel without having integration as the default time series strategy.
+You can enable writing to any time series integration plugin in parallel without having integration as the default time series strategy by enabling the `opennms-timeseries-api` feature.
+This will direct writes to both your `org.opennms.timeseries.strategy` destination and any installed timeseries integration plugin.
```

Comment:
We need to clarify this, I think:
> This will direct writes to both your `org.opennms.timeseries.strategy` destination and any installed timeseries integration plugin.

We need to be clear in the dual-write scenario, `org.opennms.timeseries.strategy` must be set to `rrd` (or not set at all, in which case the default is `rrd`) the dual-writes to the integration strategy to work. When `opennms-timeseries-api` is loaded, the secondary write to the integration strategy happens implicitly.